### PR TITLE
prov/gni: Update comments in __gnix_vc_push_work_reqs

### LINKED ISSUE #1267: Report an error when a work reg's work_fn does not return FI_SUCCESS
- the rdm_api/tsend test is failing at setup/teardown.

An error should be reported somewhere. See the TODO in `__gnix_vc_push_work_regs`.

### PR DESCRIPTION
**Pull request by e-harvey**
- Updated comments and whitespace.

### CODE REVIEW

**Comment by e-harvey** [comment]
@ztiffany Please review when you have time.

**Comment by sungeunchoi** [comment]
We discussed this in the stand up.  This is not the right fix.  That said, we should keep the formatting and spelling fixes and update the TODO comment on the call to GNIX_FATAL.  Something like, "TODO: somehow return an error to the user (e.g., via a cq error).  This situation can occur if we are try to use a resource that has already been closed.  This will most likely occur at teardown.  We can't do anything about it here, but it should be handled elsewhere."

**Review by sungeunchoi — CHANGES_REQUESTED** [review]
See my comments.

File: prov/gni/src/gnix_vc.c [hunk 1]
```diff
@@ -1787,7 +1787,7 @@ static int __gnix_vc_push_work_reqs(struct gnix_vc *vc)
 			 * due to a lack of SMSG credits. */
 			if ((ret != -FI_ENOSPC) &&
 			    (ret != -FI_EAGAIN)) {
-				/* TODO report error? */
+				/* TODO: Report error (via CQ err?) */
```
Comment (sungeunchoi) [inline prov/gni/src/gnix_vc.c]: Please add that we can't do that at this location in the code.

**Comment by sungeunchoi** [comment]
Also, title should be updated, and commits squashed with appropriately updated commit message.

**Review by sungeunchoi — APPROVED** [review]

**Pull request merged**